reporting/reports.py

```diff
@@ -100,7 +101,7 @@ def _set(self, tag, value):
                                         self.column_descriptions[c] = description
                                 i = i + 1
                 elif tag == 'params':
-                        lines = filter(lambda x: x != '', re.split('\s*\n\s*', value))
+                        lines = [x for x in re.split('\s*\n\s*', value) if x != '']
```

Comment:
I would not do that, TBH. Instead jst this:

```python
lines = filter(None, re.split(.............))
```

Reply:
See my previous comment

Reply:
Ditto.